### PR TITLE
Create React Todo App

### PR DESCRIPTION
**Pull request by yashodhicy**
🙆🏻 Create React app using vite
🙆🏻install linters
🙆🏻Add input todo form
🙆🏻Add todolist items
🙆🏻Add edit and delete buttons
🙆🏻Add styles 
🙆🏻Use SVG icons
🙆🏻fix linters
🙆🏻add Readme file

### CODE REVIEW

**Comment by HafizMassamTabraizKhan** [comment]
Hi @yashodhicy, good job so far on the ReactTodoApp project. Below are some ideas and suggestions about the project, cheers! 👍🏽

HIGHLIGHTS 🎯

Good use of props and states. ✅ 
Good use of hooks. ✅ 
Good use of Routing and components as well. ✅ 
The project is well structured and organized into folders/directories ✅
The project has comprehensive README documentation ✅
The project is very readable and easy to maintain due to the descriptive comments in your files 🥇

RECOMMENDATIONS 🤔

continue working wholeheartedly on your project

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import './styles/App.css';
+import Header from './components/Header';
+import TodosLogic from './components/TodosLogic'
+
+
+
```
Comment (mosidrum) [inline src/App.jsx:6]: - [ OPTIONAL] Avoiding keeping more than a single space throughout your code base

**Review by mosidrum — APPROVED** [review]
Hi @yashodhicy  👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Gitflow ✔️ 
Task added with the `add button` ✔️
LocalStorage ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------

File: src/App.jsx (hunk 1 above)
Comment (yashodhicy) [reply src/App.jsx:6]: Thank you

**Pull request merged**